repo/packages/A/artifactory/3/config.json

```diff
@@ -0,0 +1,203 @@
+      "type": "object",
+      "properties": {
+        "pinned-hostname": {
+          "description": "A DC/OS agent hostname to run this Artifactory instance on (e.g. 10.0.0.1). This must be specified if using local volumes with Artifactory Pro. If this is not specified then the task may relaunch on another node than the one it originally ran on.",
+          "type": "string"
+        }
+      }
+    },
+    "enterprise": {
+      "description": "Configuration for Artifactory Enterprise (a highly-available deployment of Artifactory). If enabled, this will override any Artifactory Pro configuration.",
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "description": "Whether to deploy Artifactory Enterprise. By default, this is false and Artifactory Enterprise is not deployed.",
+          "type": "boolean",
+          "default": false
+        },
+        "network": {
+          "description": "The network on which the Artifactory instances will communicate, in CIDR notation. For example: 192.168.65.0/24",
+          "type": "string"
+        },
+        "secondary": {
+          "description": "Configuration properties for the Artifactory Enterprise secondary node.",
+          "type": "object",
+          "properties": {
+            "enabled": {
+              "description": "Whether this DC/OS service represents the Artifactory Enterprise secondary instance. To run Artifactory Enterprise, you must deploy one primary DC/OS service and one secondary DC/OS service. By default, this is false (i.e. a primary instance).",
+              "type": "boolean",
+              "default": false
+            },
+            "unique-nodes": {
+              "description": "Run secondary instances on unique hosts. By default, this is true.",
+              "type": "boolean",
+              "default": true
+            },
+            "nodes": {
+              "description": "Number of Artifactory Enterprise secondary instances. By default, this is 1.",
+              "type": "integer",
+              "default": 1
+            },
+            "name": {
+              "description": "The DC/OS service name of the primary Artifactory Enterprise instance to route requests to.",
+              "type": "string",
+              "default": "artifactory"
+            },
+            "api-key": {
+              "description": "API key generated by the primary Artifactory. This is necessary for the secondary instances to authenticate against the primary instance.",
+              "type": "string"
+            }
+          },
+          "required": [
+            "enabled"
+          ]
+        }
+      },
+      "required": [
+        "enabled"
+      ]
+    },
+    "database": {
+      "description": "Database properties.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Database type. (can be MySQL, Oracle, MS SQL Server or Postgres)",
```

Comment:
Minor, but: Database type (can be MySQL, Oracle, MS SQL Server or Postgres)*.*

Reply:
(i.e., move the `.`)

Reply:
Got it.